content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_entry.md

```diff
@@ -3,8 +3,6 @@
 
 The `ledger_entry` method returns a single ledger object from the XRP Ledger in its raw format. See [ledger format][] for information on the different types of objects you can retrieve.
 
-**Note:** There is no commandline version of this method. You can use the [json method][] to access this method from the commandline instead.
```

Comment:
This note remains true. The `json` method is a workaround you can use with any method but it's not commandline syntax for this method.